packages/elements/src/tree/helpers/renderer.ts

```diff
@@ -30,12 +30,12 @@ export const createTreeRenderer = <T extends TreeDataItem = TreeDataItem>(
     element: HTMLElement = document.createElement('ef-tree-item')
   ): HTMLElement => {
     // cast type to element to not break List api.
-    const _context = context as RendererScope<T>;
+    const _context = context as RendererScope<T> | undefined;
```

Comment:
What is the case that context is undefined?

Reply:
`context` is an optional parameter.